Simulation/generate_job_files.jl

```diff
@@ -33,7 +33,7 @@ for taskfiles in Iterators.partition(task_files, ntasks_per_job)
     for f in taskfiles
         tasks *= "srun -n 1 --exclusive $f &\n"
     end
-    tasks = chopsuffix(tasks, "&\n")
+#     tasks = chopsuffix(tasks, "&\n")
```

Comment:
I think this should not be commented, do you have a reason for that? I remember from past experience in other cluster that the & must indeed not be there
Following ccin2p3 documentation, it follows the same approach: [multiple-tasks-script](https://doc.cc.in2p3.fr/en/Computing/slurm/examples.html#multiple-tasks-script)

Reply:
Fixed when using julia 1.10 (added in README)